Admin/AbstractAdmin.php

```diff
@@ -2729,61 +2729,61 @@ public function hasAccess($action, $object = null)
     /**
      * {@inheritdoc}
      */
-    public function configureActionButtons($action, $object = null)
+    public function configureActionButtons($actionList, $action, $object = null)
```

Comment:
Because this method allow you to configure the button**s** on the top right for a current action (e.g. show, edit, list, ...).

You need the `action` and `object` parameter to detect the current action and the `actionList` to store all buttons.